content/influxdb/v2.4/upgrade/v1-to-v2/automatic-upgrade.md

```diff
@@ -174,7 +175,20 @@ InfluxDB {{< current-version >}} doesn't support [in-memory (`inmem`) indexing](
 
 ### Interactive shell
 
-The InfluxDB 2.x `influx` CLI does not include an interactive shell for executing InfluxQL queries or Flux queries. For information about executing InfluxQL queries, see [Query data with InfluxQL](/influxdb/v2.4/query-data/influxql/). To build an interactive shell to execute Flux queries, [compile and build a command line Flux REPL from source](https://github.com/influxdata/flux/blob/master/README.md#getting-started). 
+The InfluxDB {{< current-version >}} `influx` CLI includes an interactive **InfluxQL shell** for executing InfluxQL queries.
+To start an InfluxQL shell:
+
+1. Download and install in the [`influx` CLI](/influxdb/v2.4/tools/influx-cli/).
+2. Set up your [`influx` CLI authentication credentials](/influxdb/v2.4/tools/influx-cli/#set-up-the-influx-cli).
+3. Run the `influx v1 shell` command.
+
+For more information see:
+
+- [Use the InfluxQL shell](/influxdb/v2.4/tools/influxql-shell/)
+- [Query data with InfluxQL](/influxdb/v2.4/query-data/influxql/)
```

Comment:
Would it make sense to include this info for manual and docker upgrades as well?

If yes, perhaps in each upgrade doc, we say something along the lines of:
After upgrading, you have the option to use an interactive shell to query data in InfluxDB. For information about querying InfluxQL, see how to use the InfluxQL shell. For information about querying Flux, see...

Thought about alternative of adding to "Upgrade from InfluxDB 1.x to 2.4" and occurs that folks may click one of the choices manual, automatic, etc, and then we'd need to link from procedure back to top-level, which felt a bit circular

Reply:
Added InfluxQL info to the other upgrade docs.